### DISCUSSION TITLE
Possibility to add external webpage to profile

### DISCUSSION THREAD

**Discussion by niklasnet** [Traewelling/traewelling#2010] [Ideas / Feature Requests]
Currently, you can just have hyperlinks to Mastodon (and Twitter if you had already a connection before the Twitter/X API changes).  

Since more alternatives to Twitter/X are circulating around, it might be a good idea to include the possibility for adding an external webpage to your profile (Linktree, your own webpage or Bluesky for ex.). 

For being certain about security, a disclaimer pop-up could be generated to remind that the user is about to leave the Trwl page and the team behind Trwl is not responsible for the content.

**Comment by niklasnet** [comment]
I've done some HTML magic, a possible option could be to just add a symbol which is referencing to the external webpage, similar to the event check-in view: 

<img width="1397" alt="image" src="https://github.com/Traewelling/traewelling/assets/28200315/9b5f488f-37c9-4412-859d-eaa6f2804bc2">

**Comment by MrKrisKrisu** [comment]
I guess I can close this discussion as you can use the bio for this.
